src/plugins/mf_priority.cpp

```diff
@@ -241,8 +241,9 @@ static int increment_resources (Association *b,
     b->cur_cores = b->cur_cores + (counts.nslots * counts.slot_size);
 
     // increment cur_nodes for queue
-    b->queue_usage[queue].cur_nodes = b->queue_usage[queue].cur_nodes +
-                                      counts.nnodes;
+    if (queue != "")
+        b->queue_usage[queue].cur_nodes = b->queue_usage[queue].cur_nodes +
+                                          counts.nnodes;
```

Comment:
Definitely not a C++ expert, but would it be more idiomatic (and potentially a little faster) to use `!queue.empty()` here?

Reply:
I think you're absolutely right, thanks for this suggestion. I'll change to just use `.empty ()`.